CodeEdit/Features/Settings/Pages/ThemeSettings/Models/ThemeModel.swift

```diff
@@ -25,9 +25,13 @@ final class ThemeModel: ObservableObject {
         filemanager.homeDirectoryForCurrentUser.appendingPathComponent("Library/Application Support/CodeEdit")
     }
 
-    /// The URL of the `themes` folder
+    internal var bundledThemesURL: URL? {
```

Comment:
As far as I know variables are internal by default, so you can remove `internal`.